File: server/models/account.js
const mongoose = require("mongoose");

const AccountSchema = new mongoose.Schema({
  accountName: {
    required: true,
    type: String,
  },
  bankName: {
    type: String,
  },
  receiver: {
    type: String,
    required: true,
  },
  iban: {
    required: true,
    type: String,
  },
  phone: {
    //to understand which drivers' Accounts they are
    type: String,
    required: true,
  },
});

module.exports = Account = mongoose.model("Account", AccountSchema);

File: socoolbus/lib/models/account.dart
class Account {
  final String accountId;
  final String accountName;
  final String bankName;
  final String receiver;
  final String iban;

  const Account(
      {required this.accountId,
      required this.accountName,
      required this.bankName,
      required this.receiver,
      required this.iban});

        @override
  String toString() {
    return accountName;
  }

  @override
  bool operator ==(dynamic other) =>
      other != null && other is Account && this.accountId == other.accountId;

  factory Account.fromJson(Map<dynamic, dynamic> json) {
    return Account(
      accountId: json['_id'],
      accountName: json['accountName'],
      bankName: json['bankName'],
      receiver: json['receiver'],
      iban: json['iban'],
    );
  }
}
